src/test/java/org/broadinstitute/hellbender/tools/exome/allelefraction/AFInitializerUnitTest.java

```diff
@@ -0,0 +1,76 @@
+package org.broadinstitute.hellbender.tools.exome.allelefraction;
+
+import htsjdk.samtools.util.Interval;
+import org.broadinstitute.hellbender.tools.exome.AllelicCount;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.*;
+
+/**
+ * Created by davidben on 11/24/15.
+ */
+public class AFInitializerUnitTest {
+    @Test
+    public void testInitialize() throws Exception {
```

Comment:
There should be no reason to have ``throws Exception`` in a unit test that I can think of.